#! /bin/sh /usr/share/dpatch/dpatch-run
## 95-build-ace-for-tao.dpatch by  <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Always define nobase_include_HEADERS, regardless of BUILD_ACE_FOR_TAO

@DPATCH@

diff -rupd ACE_wrappers.orig/ace/Makefile.am ACE_wrappers/ace/Makefile.am
--- ACE_wrappers.orig/ace/Makefile.am	2009-11-05 22:47:49.000000000 +0100
+++ ACE_wrappers/ace/Makefile.am	2009-11-06 00:35:29.000000000 +0100
@@ -27,7 +27,7 @@ nobase_include_HEADERS =
 
 ## Makefile.ACE.am
 
-if !BUILD_ACE_FOR_TAO
+## if !BUILD_ACE_FOR_TAO
 
 lib_LTLIBRARIES += libACE.la
 
@@ -1261,7 +1261,7 @@ CLEANFILES += \
 ACE.pc: ${top_builddir}/config.status ${srcdir}/ACE.pc.in
 	${top_builddir}/config.status --file $@:${srcdir}/ACE.pc.in
 
-endif !BUILD_ACE_FOR_TAO
+## endif !BUILD_ACE_FOR_TAO
 
 EXTRA_DIST = \
   ACE.pc.in \
